src/backend/db/products.js

```diff
@@ -125,7 +125,8 @@ export const products = [
     onSale: false,
     starRatings: 5,
     mountType: "Wall",
-    color: "blue"
+    color: "blue",
+    qty: 1
```

Comment:
Just a suggestion. you can always share the context in pr description if you have changed anything other than that is needed.